command/agent/cache/lease_cache.go

```diff
@@ -274,7 +274,7 @@ func (c *LeaseCache) Send(ctx context.Context, req *SendRequest) (*SendResponse,
 		return cachedResp, nil
 	}
 
-	c.logger.Debug("forwarding request", "method", req.Request.Method, "path", req.Request.URL.Path)
+	c.logger.Info("forwarding request from cache", "method", req.Request.Method, "path", req.Request.URL.Path)
```

Comment:
I would be wary about making something like this (which fires on nearly every request handled, ideally) be Info level.

Reply:
We have an almost identical log that does this already - here's an excerpt from the logs. The top one is the one I changed, and the second is a pre-existing log I didn't change:
```
2022-09-01T14:59:47.927Z [INFO]  cache.leasecache: forwarding request from cache: method=GET path=/v1/secret/my-secret
2022-09-01T14:59:47.927Z [INFO]  cache.apiproxy: forwarding request: method=GET path=/v1/secret/my-secret
```

Reply:
It felt strange that they were on differing log levels. I'm not averse to moving them both to debug, but I think info makes sense too.

Reply:
To me it makes sense that they're at different levels.  The apiproxy line is already telling the reader the main event, that we're reaching out to Vault in order to service a request. The leasecache line is giving additional info, namely that the cache was involved in service this request.  I don't think we need two lines at Info level every time forwarding occurs, do you?

Reply:
I feel like it is useful to know that the request has been forwarded from the cache layer to the API proxy later, in case anything goes wrong between the two. For additional context, if caching is enabled, both of these lines will be present, and if it's not, neither will be. I do feel like if one of them deserves to be Info, then both do, and I'll admit I'd be more eager to move them both to debug rather than have them differ. 

To me, it's useful information to see that since the request wasn't in the cache, we're forwarding the request, perhaps more so than the second log made just to indicate we're making the request (it's less of a 'forward', in that case, as the next line is making the request).

All this being said, this issue isn't the most burning issue in my life and I'll live no matter what the log levels are. How about an alternative that remains with one log: we keep the leasecache log as info, but move the apiproxy one down to debug?

Reply:
You can have the proxy without the cache, and then you'd get no log at Info.

Reply:
I'll change the first one - then, like I said, it's not a hill I'm willing to die on and I can definitely understand only wanting one log. Thanks for talking it through!

Reply:
I've set the first to Debug and added a small amount more info to the second. Thanks again!